openedx/core/djangoapps/programs/utils.py

```diff
@@ -280,7 +276,10 @@ def _extend_course_runs(self):
                 self._execute('_attach_course_run', course_run)
 
     def _attach_course_run_certificate_url(self, run_mode):
-        certificate_data = certificate_api.certificate_downloadable_status(self.user, self.course_run_key)
```

Comment:
@rlucioni Please confirm whether or not you think we should revert this change and move the user check to the subclass.